### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by xcodz-dot**
Needed to tell people so we do not receive any duplicate solution. Do not count this as hactoberfest-accepted

### **Describe your change:**

* [ ] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [x] Documentation change?

### **Checklist:**
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [ ] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [ ] All new Python files are placed inside an existing directory.
* [ ] All filenames are in all lowercase characters with no spaces or dashes.
* [ ] All functions and variable names follow Python naming conventions.
* [ ] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [ ] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [ ] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

**Comment by xcodz-dot** [comment]
@dhruvmanila @poyea, check this (ready to merge)

**Review by poyea — APPROVED** [review]
You may bold `same implementation`. Let's wait for possibly one more approval. Thanks.

File: CONTRIBUTING.md
Comment (xcodz-dot) [inline CONTRIBUTING.md:18]:
```suggestion
**New implementation** is welcome! For example, new solutions for a problem, different representations for a graph data structure or algorithm designs with different complexity but **same implementation** of a already existing implementation is not allowed. Please check whether the solution is already implemented or not before submitting you pull request.
```

File: CONTRIBUTING.md
Comment (shellhub) [inline CONTRIBUTING.md:18]:
```suggestion
**New implementation** is welcome! For example, new solutions for a problem, different representations for a graph data structure or algorithm designs with different complexity but **same implementation** of a already existing implementation is not allowed. Please check whether the solution is already implemented or not before submitting your pull request.
```

**Comment by xcodz-dot** [comment]
@shellhub check this out

**Pull request merged**
